server/helpers/passErrors.js

```diff
@@ -0,0 +1,9 @@
+export default function passErrors(controller) {
```

Comment:
I understand why this was done and it's because we (namely me - Elijah) did a poor job of handing off errors in the controller. I'm wondering if it's not too late to update the controllers to use next() in passing off errors to subsequent handlers. This approach is also great IMO but I know it's done differently in the rest of our microservices

Reply:
This is definitely different. If we want to conform, we can just `return next(err)` in the controllers rather than `throw err`. I personally prefer the latter, but the former is more explicit in what's actually happening for sure.